src/components/TimeRangeSelect/index.js

```diff
@@ -9,29 +9,23 @@ type Props = {
   options: Array<{ value: number, label: string }>
 };
 
-const TimeRangeSelect = (props: Props) => {
-  function handleChange(e) {
-    props.selectedTimeRange(e.target.value);
-  }
-
-  return (
-    <div className="time-range-select">
-      <div className="time-range-select__field">
-        <label className="time-range-select__label" htmlFor="timeRange">
-          Time Range:{' '}
-        </label>
-        <Field
-          className="time-range-select__dropdown"
-          name="timeRange"
-          options={props.options}
-          component={Select}
-          onChange={handleChange}
-        />
-      </div>
+let TimeRangeSelect = ({ selectedTimeRange, options }: Props) => (
```

Comment:
Just curious, is this line equivalent to:
```let TimeRangeSelect = ({ selectedTimeRange, options }) => (```
(The destructure syntax in ES6+ confuses me from time to time).

Reply:
Yes those are equivalent. It took me a while to get used to it too.

Reply:
Thanks. I am going to approve this PR now. Please feel free to close it after resolving the conflicts.

Reply:
Thanks! I'll merge it after the one in the backend.

Reply:
By the way, since they are equivalent, do you mind removing `: Props` part from that line? I am afraid the next time when I see this line, I would still ask myself what it means.

Reply:
I forgot that `Props` is the type annotation. You can leave it as is.